pkg/cluster/cluster.go

```diff
@@ -59,10 +56,11 @@ type Spec struct {
 	CustomProperties map[string]string
 }
 
-type AWSCredentials struct {
-	AccountID       string
-	AccessKeyID     string
-	SecretAccessKey string
+type CCS struct {
```

Comment:
we are going to support also GCP CCS so I prefer we won't go with this direction. we should keep `AWSCredentials` struct

Reply:
I wanted a single struct for all CCS needs.
Is it better if I nest AWSCredentials struct inside CCS rather than flat?

Reply:
+1

Reply:
done, PTAL